openlibrary/templates/batch_import.html

```diff
@@ -0,0 +1,37 @@
+$def with (batch_result=None)
+$# type: (BatchResult)
+
+<div class="batchImport">
+  <div id="contentBody">
+    <h1>$_('Batch Imports')</h1>
+    <p>$_('Attach a JSONL formatted document with import records.')</p>
+    <p>$:_('See the <a href=\"https://github.com/internetarchive/openlibrary-client/tree/master/olclient/schemata\">olclient import schemata</a> for more.')</p>
+    <form method="POST" enctype="multipart/form-data" action="">
+      <input type="file" name="batchImport" />
+      <br/>
+      <input type="submit" value="$_('Import')"/>
```

Comment:
Small styling tweaks

```suggestion
      <hr/>
      <button type="submit">$_('Import')</button>
```